doc/content/devices/adding-devices/_index.md

```diff
@@ -17,6 +17,8 @@ This section contains instructions for adding LoRaWAN® devices in {{% tts %}}.
 End devices in {{% tts %}} are managed under **Applications**.
 An application is a logical collection of devices that can be used to collect devices by function or geographical area. Before proceeding with this guide, [create an application]({{< ref "/integrations/adding-applications/" >}}) first.
 
+{{< note >}} In {{% tts %}} Cloud tenant, there's no restriction on registering number of devices within an application. This means you can register all of your devices under a single application without any data routing issues. For efficient management, group devices and register in multiple applications based on deployment sites or clients. This helps to streamline permissions and manage organizations, especially when handling multiple clients. {{</ note >}}
```

Comment:
I think this is not a place for this. We have a whole section dedicated to TTS Cloud https://www.thethingsindustries.com/docs/the-things-stack/cloud/, so if anywhere, this should be placed there